python/ray/tests/test_runtime_env.py

```diff
@@ -131,6 +138,24 @@ def test_two_node_module(two_node_cluster, working_dir, client_mode):
     assert len(list(Path(PKG_DIR).iterdir())) == 1
 
 
+@unittest.skipIf(sys.platform == "win32", "Fail to create temp dir.")
+@pytest.mark.parametrize("client_mode", [True, False])
+def test_two_node_local_file(two_node_cluster, working_dir, client_mode):
+    with open(os.path.join(working_dir, "test_file"), "w") as f:
+        f.write("1")
+    cluster, _ = two_node_cluster
+    (address, env, PKG_DIR) = start_client_server(cluster, client_mode)
+    runtime_env = f"""{{  "working_dir": "{working_dir}" }}"""
+    execute_statement = """
+vals = ray.get([check_file.remote('test_file')] * 1000)
+print(sum([int(v) for v in vals]))
+"""
+    script = driver_script.format(**locals())
+    out = run_string_as_driver(script, env)
+    assert out.strip().split()[-1] == "1000"
```

Comment:
please add a comment for what this is doing; it's a bit hard to parse for an unfamiliar reader